include/variant_parser/variant_record.hpp

```diff
@@ -220,4 +224,6 @@ class variant_record
     float qual{};
     std::string filter{"PASS"};
     std::map<std::string, std::string> info{};
+    std::string format{"GT"};
+    std::string mysample{"./."};
```

Comment:
I would suggest to rename this field from `mysample` to `genotype` because that's what it contains. `mysample` is just a placeholder for the real sample name that the user can provide via the CLI.